starter-apps/vue/vue-template/src/components/forms/TextArea.vue

```diff
@@ -0,0 +1,19 @@
+<script setup>
+const { id, name, label, required } = defineProps([
+  'id',
+  'name',
+  'label',
+  'placeholder',
+  'required'
+])
```

Comment:
Same comment: This seems to be missing some important props like `hint` and `error-message` for example.

Reply:
Same comment as above